.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+  test:
+    runs-on: macos-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        module:
+          - rxbinding
+          - rxbinding-appcompat
+          - rxbinding-core
+          - rxbinding-drawerlayout
+          - rxbinding-leanback
+          - rxbinding-material
+          - rxbinding-recyclerview
+          - rxbinding-slidingpanelayout
+          - rxbinding-swiperefreshlayout
+          - rxbinding-viewpager
+          - rxbinding-viewpager2
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Clean Cache
+        run: mv ~/.gradle/caches ~/.gradle/.invalid_caches
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-${{ hashFiles('**/build.gradle') }}-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+
+      - name: Run Tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 18
```

Comment:
Eventually we'll want to shard this to include minSdk and the latest. And maybe a random value in-between.